spec/lib/granite/rspec/satisfy_preconditions_spec.rb

```diff
@@ -26,6 +26,10 @@
 
     specify { expect(action).not_to satisfy_preconditions.with_message('Precondition failed') }
 
+    specify { expect(action).not_to satisfy_preconditions.with_message(/failed$/) }
+
+    specify { expect(action).not_to satisfy_preconditions.with_messages(['Precondition failed', /failed$/, /^Precondition/]) }
```

Comment:
Wondering why does it expect an array rather than a list of arguments.

Reply:
Yup, there is an inconsistency with `messages_of_kind`. I will update to support both